### DISCUSSION TITLE
i3 window tabs

### DISCUSSION THREAD

**Discussion by iliesh** [hyprwm/Hyprland#4786] [Q&A]
Another i3 user here trying to migrate to Hyprland but stuck on the tab/stack view. togglegroup - will be something that help - but it's not fully functional as i3 does, is it possible to create a similar action, like togglegroupall - which will add to a group all current windows from the active screen?

**Comment by vaxerski** [comment]
I believe only via plugins, tho it's a reasonable FR. Feel free to make a MR.
